src/v2i-hub/CARMAStreetsPlugin/src/CARMAStreetsPlugin.cpp

```diff
@@ -286,6 +288,26 @@ void CARMAStreetsPlugin::HandleMobilityPathMessage(tsm2Message &msg, routeable_m
 	}
 }
 
+void CARMAStreetsPlugin::HandleSRMMessage(SrmMessage &msg, routeable_message &routeableMsg)
+{
+	J2735ToSRMJsonConverter srmJsonConverter;
```

Comment:
Can we add something for error handling. Like updating some count or something and catching some possible exceptions to avoid plugin crashes on bad messages.

Reply:
https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/518/commits/b63bfbec39f57ad20a45bff5d8c0595b69cc64e9